readme.md

```diff
@@ -2,7 +2,18 @@
 
 ![GitHub Repo stars](https://img.shields.io/github/stars/jellyfish-dev/jellyfish-dashboard)
 
-An internal tool used to showcase [Jellyfish](https://github.com/jellyfish-dev/jellyfish)'s capabilities.
+## What is Jellyfish Dashboard?
```

Comment:
Consider renaming the first header from `Jellyfish React Client Demo - Dashboard`  to `Jellyfish Dashboard`